client-base/src/main/kotlin/app/cash/backfila/client/RealBackfillModule.kt

```diff
@@ -21,8 +21,8 @@ class RealBackfillModule(
   constructor(
     config: BackfilaClientConfig,
     loggingSetupProvider: KClass<out BackfilaClientLoggingSetupProvider> =
-        BackfilaClientNoLoggingSetupProvider::class
-  ) : this( Provider { config }, loggingSetupProvider)
+      BackfilaClientNoLoggingSetupProvider::class
```

Comment:
Nit: remove whitespace changes from PR so you don't end up on a `git blame` by accident